src/server/src/system/SystemAttack.cpp

```diff
@@ -222,7 +222,7 @@ void SubSystemAttack::Attack(anax::Entity attacker)
 			if (damage != 0) {
 				std::cout << "hit entity " << entity.getId().index << " for " << damage
 				          << std::endl;
-				health.HealthUpdate(-1 * damage);
+				health.HealthUpdate(-10 * damage);
```

Comment:
This shouldn't be done. If you want to do more damage then change the weapons. This is just going to cause confusion later

Reply:
It also breaks the printing, because it's just a terrible hack